pint/facets/numpy/numpy_func.py

```diff
@@ -965,7 +965,7 @@ def implementation(a, *args, **kwargs):
         return a._REGISTRY.Quantity(func(a_stripped, *args, **kwargs))
 
 
-for func_str in ("cumprod", "cumproduct", "nancumprod"):
+for func_str in ("cumprod", "cumprod", "nancumprod"):
```

Comment:
```suggestion
for func_str in ("cumprod", "nancumprod"):
```

Reply:
thanks, I totally missed that `cumprod` was already there.